app/assets/javascripts/validate.js

```diff
@@ -0,0 +1,50 @@
+//= require jquery
+//= require jquery_ujs
+//= require turbolinks 
+//= require_tree .
+
+var userName = document.getElementById('username');                            // Declare and set elements to variables.
+var userEmail = document.getElementById('email');
+var userMsg = document.getElementById('message');
+var valMessageName = document.getElementById('message-name');
```

Comment:
why not just have one div, and append each error message? or better yet, append the error message below the input?